v03/evaluate.py

```diff
@@ -56,33 +72,48 @@ class GraphEvaluateDefinition(EvaluateDefinition):
     Edit notes (Sam): TF 2.0 has a tf.function class that builds computational graphs automatically (is recommended), see operators.py
     '''
     @abstractmethod
-    def build_graph(self):
+    def evaluate(self):
         pass
 
+    """
+    Rodd_layout --conversion steps--> Graph Layout
+    Graph:
+        Nodes:
+            Inputs: [Input Nodes]
+            Args: [Arg Types]
+            Function: [Dense, ResNet]
+        Edges:
+    """
+
     @abstractmethod
-    def reset_graph(self):
+    def build_graph(self):  # fill Genome?
+        """
+        This will fill the genome and the arguments. Randomize connections.
+        Original code is in genome.py. FillArgs and
+        """
         pass
 
     @abstractmethod
-    def train_graph(self):
+    def reset_graph(self):  # resetAttrVals
+        """
+        Original code in genome.py's resetEvalAttr function.
+        """
         pass
 
-    @abstractmethod
     def run_graph(self):
         pass
 
 
 
-
 class IndividualStandardEvaluate(EvaluateDefinition):
     def __init__(self):
         pass
 
-    def evaluate(self, indiv_def, indiv, training_datapair, validation_datapair=None):
+    def evaluate(self, indiv_def, indiv, data: ezData):
         for block_index, block in enumerate(indiv.blocks):
             block_def = indiv_def[block_index]
             if block.need_evaluate:
-                training_datapair = indiv_def[block_index].evaluate(block_def, block, training_datapair, validation_datapair)
+                training_datapair = indiv_def[block_index].evaluate(block_def, block, data)
```

Comment:
if we are replacing training_datapair + validation_datapair with 'data' then it should also return 'data'
```suggestion
                data = indiv_def[block_index].evaluate(block_def, block, data)
```